server/graphql/types.js

```diff
@@ -428,8 +434,8 @@ export const ResponseType = new GraphQLObjectType({
       },
       user: {
         type: UserType,
-        resolve(response) {
-          return response.getUser();
+        resolve(response, args, req) {
+          return response.getUserForViewer(req.remoteUser);
```

Comment:
this still doesn't feel right. We'd have to implement this same logic on every type that can get a user... Is there no way to implement this once in the `email` field of `UserType`?